lti_consumer/tests/unit/plugin/test_views.py

```diff
@@ -375,6 +409,46 @@ def test_callback_endpoint_dl_content_launch_database_config(self, dl_value, moc
         # Check response
         self.assertEqual(response.status_code, 200)
 
+    @patch('lti_consumer.lti_1p3.consumer.LtiConsumer1p3.set_custom_parameters')
+    @patch('lti_consumer.plugin.views.parse_custom_parameters')
+    def test_launch_existing_custom_parameters(self, mock_set_custom_parameters, mock_parse_custom_parameters):
```

Comment:
Could you please `mock_parse_custom_parameters` was called with appropiate arguments?. @kuipumu